params/default.go

```diff
@@ -14,8 +15,47 @@ func DefaultNetwork() Network {
 }
 
 func init() {
+	// check if a different network from the registry was specified
+	if network, ok := os.LookupEnv("CELESTIA_NETWORK"); ok {
+		if _, exists := networksList[Network(network)]; !exists {
+			panic("unknown network specified")
+		}
+		defaultNetwork = Network(network)
+		return
+	}
+	// check if private network option set
 	if genesis, ok := os.LookupEnv("CELESTIA_PRIVATE_GENESIS"); ok {
 		defaultNetwork = Private
 		genesisList[Private] = strings.ToUpper(genesis)
 	}
+	// check if custom network option set
+	if customNet, ok := os.LookupEnv("CELESTIA_CUSTOM"); ok {
+		fmt.Print("\n\nWARNING: Celestia custom network specified. Only use this option if the node is " +
+			"freshly created and initialized.\n**DO NOT** run a custom network over an already-existing node " +
+			"store!\n\n")
+
+		params := strings.Split(customNet, "=")
+		// ensure both params are present
+		if len(params) != 2 {
+			panic("must provide CELESTIA_CUSTOM in this format: <network_ID>=<genesis hash>")
+		}
+		netID, genHash := params[0], params[1]
+
+		defaultNetwork = Network(netID)
+		// register new network and set as default for node to use
+		networksList[defaultNetwork] = struct{}{}
+		genesisList[defaultNetwork] = strings.ToUpper(genHash)
+	}
```

Comment:
CELESTIA_CUSTOM should before CELESTIA_NETWORK so that it adds to the registry and the forme reads. Otherwise it will fail